### ISSUE TITLE
Undefined variable: b:doge_supported_doc_standards

### ISSUE THREAD

**Issue by bstaint** [kkoomen/vim-doge#80]
**Describe the bug**
```
Error detected while processing function doge#generate:
line    2:
E121: Undefined variable: b:doge_supported_doc_standards
E116: Invalid arguments for function index(b:doge_supported_doc_standards, b:doge_doc_standard) < 0
```

**Settings**

**To Reproduce**

1. open unsupported file(1.vim), press `<Leader>d`

**Expected behavior**

**Screenshots**

**Out of `vim --version`**

**Comment by kkoomen** [comment]
This bug has been fixed and released in [v2.2.3](https://github.com/kkoomen/vim-doge/tree/v2.2.3).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.

**Comment by bstaint** [comment]
Tks.

Is it repeat?
https://github.com/kkoomen/vim-doge/blob/50efe22af6b677acc7d5645da75a5a47b056b729/autoload/doge.vim#L31
